Configurations/10-main.conf

```diff
@@ -934,28 +934,28 @@ sub vms_info {
     # in order to build "universal" binary and allow OpenSSL take
     # advantage of NEON when it's available.
     #
-    "android-armeabi" => {
-        inherit_from     => [ "android", asm("armv4_asm") ],
+    "android-armeabi-base" => {
+        inherit_from     => [ asm("armv4_asm") ],
     },
-    "android-mips" => {
-        inherit_from     => [ "android", asm("mips32_asm") ],
+    "android-mips-base" => {
+        inherit_from     => [ asm("mips32_asm") ],
         perlasm_scheme   => "o32",
     },
 
-    "android64" => {
+    "android64-base" => {
         inherit_from     => [ "linux-generic64" ],
-        cflags           => add(picker(default => "-mandroid -fPIC --sysroot=\$(CROSS_SYSROOT) -Wa,--noexecstack")),
+        cflags           => add(picker(default => "-fPIC --sysroot=\$(CROSS_SYSROOT) -Wa,--noexecstack")),
         bin_cflags       => "-pie",
     },
-    "android64-aarch64" => {
-        inherit_from     => [ "android64", asm("aarch64_asm") ],
+    "android64-aarch64-base" => {
+        inherit_from     => [ asm("aarch64_asm") ],
         perlasm_scheme   => "linux64",
     },
-    "android64-x86_64" => {
-        inherit_from     => [ "android64", asm("x86_64_asm") ],
+    "android64-x86_64-base" => {
+        inherit_from     => [ asm("x86_64_asm") ],
```

Comment:
There actually is a bug here. Not contributor's fault! It was there all along. One has to omit RC4_CHAR inherited from linux-generic64...